src/jsonpath-browser.js

```diff
@@ -81,6 +91,10 @@ JSONPath.prototype.vm = {
         // eslint-disable-next-line no-new-func
         return (new Function(...keys, code))(...values);
     }
+}
+
+JSONPath.prototype.vm = {
+    Script
```

Comment:
These changes are to match the new Script creation, then execution. There might be room to perform similar pre-compiling here for browser environments, but that's outside the scope of this PR.